adaptivemd/scheduler.py

```diff
@@ -362,7 +362,7 @@ def replace_prefix(self, path):
         path = path.replace('sandbox://', '../..')
 
         # the main remote shared FS
-        path = path.replace('shared://', '../../..')
+        path = path.replace('shared://', '')
```

Comment:
This is indeed a very bad hack, if not more a bug. I actually doubt that this will work. What it does it that everywhere, where you expect from the working directory to link to `NO_BACKUP` you will end up in the working directory instead. Possible that it still works because it is not used yet.

You can use `worker://` instead because that is actually what worker does. Just don't alter the path and if you use an absolute path then this works. 

```
worker:///this/is/an/absolute/path/traj.dcd`
```

note the 3! `/` in the beginning, while relative paths in the working dir start with only 2 `/`.